scripts/add-icon.js

```diff
@@ -0,0 +1,71 @@
+import fs from 'node:fs/promises';
+import inquirer from 'inquirer';
+import chalkPipe from 'chalk-pipe';
+import logSymbols from 'log-symbols';
+import getRelativeLuminance from 'get-relative-luminance';
+import { getIconsDataString, getIconDataPath, titleToSlug } from './utils.js';
+
+const hexPattern = /^#?[a-fA-F0-9]{6}$/;
+const iconsData = JSON.parse(await getIconsDataString());
+
+const prompts = [
+  {
+    type: 'input',
+    name: 'title',
+    validate: (text) => {
```

Comment:
What happens if a title generates an existent icon slug? That must be validated.

Reply:
Yep. Will use the same logic as `our-lint`.